spec/models/cafe_category_data_sql_view_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+RSpec.describe CafeCategoryData do
```

Comment:
Used this test to verify the data against a smaller sample size that I could calculate the results of